mmpose/evaluation/metrics/keypoint_2d_metrics.py

```diff
@@ -77,9 +77,9 @@ def process(self, data_batch: Sequence[dict],
             # ground truth data_info
             gt = data['data_sample']['gt_instances']
             # ground truth keypoints coordinates, [1, K, D]
-            gt_coords = gt['ori_keypoints']
+            gt_coords = gt['keypoints']
             # ground truth keypoints_visible, [1, K, 1]
-            mask = gt['ori_keypoints_visible'].astype(bool).reshape(1, -1)
+            mask = gt['keypoints_visible'].astype(bool).reshape(1, -1)
```

Comment:
I think it is more reasonable to convert `mask` to boolean array here instead of in `_calc_distances`. This matches intuition and descriptions better. This conversion should also be applied to `mask` in the other Metrics.